### ISSUE TITLE
Can't Install Yarn into JS

### ISSUE THREAD

**Issue by Deliceli58** [metaplex-foundation/metaplex#1684]
**I tried much but I can't install that**
When I run the command that yarn install it gives that error and Im stuck in there please help me Im already have yarn version 1.22.17

while ım trying to solve my problem my errors are change to that and ''yarn build'' code's error process become ''yarn''s error process everything is in photos.
when I say yarn VS code gives me error that uptake thing

<img width="1440" alt="yarn build" src="https://user-images.githubusercontent.com/98783173/151885880-97691e04-18b0-4918-96eb-e4861a7f0734.png">
<img width="1440" alt="yarn1" src="https://user-images.githubusercontent.com/98783173/151885902-8098282e-b5b9-4c27-8ddc-da74d766101f.png">
<img width="1440" alt="yarn2" src="https://user-images.githubusercontent.com/98783173/151885919-acab38d4-0105-4502-8f0d-10a48316206a.png">
<img width="1440" alt="yarn3" src="https://user-images.githubusercontent.com/98783173/151885926-f4b6a1b4-98d5-48ad-abe6-2409292f0103.png">

**Comment by BORRODA8** [comment]
try to reinstall yarn on your pc. It's helps me

**Comment by Deliceli58** [comment]
> try to reinstall yarn on your pc. It's helps me

how can I do that can you write me the exact queue of commands for that please I'm really newbie in coding

**Comment by Khushhal23** [comment]
@Deliceli58 Run npm install -g yarn or run npm install yarn in your metaplex folder

**Comment by stegaBOB** [comment]
This issue has been fixed as of #1673.
